.travis.yml

```diff
@@ -32,6 +32,8 @@ matrix:
         - cargo test --verbose -p sgx-isa --features sgxstd -Z package-features --target x86_64-fortanix-unknown-sgx --no-run
         - cargo test --verbose -p sgxs-tools --features pe2sgxs --bin isgx-pe2sgx -Z package-features
         - cargo test --verbose -p dcap-ql --features link -Z package-features
+        - cargo build --verbose -p aesm-client
```

Comment:
I'm confused, the sgxs feature is already not set on line 31. Why do we need to test this separately?

Reply:
:man_facepalming: 
My mistake. I overlooked that somehow